### DISCUSSION TITLE
How to display the original image cut in dzi format in my local folder

### DISCUSSION THREAD

**Discussion by wdj577** [MIDA-group/CytoBrowser#183] [Q&A] [answered]
I used [](https://openseadragon.github.io/examples/creating-zooming-images/) to cut an original image in dzi format, how to display it? Can the configuration steps be more clear?

**Comment by joakimlindblad** [comment, marked as answer]
So, I assume that you followed the instructions and ran
```
npm install
node cytobrowser.js 127.0.0.1 8080
```
and upon opening http://localhost:8080/
you get the message: _"The server was not able to find any images in the data directory. Make sure it's populated and try again."_

For the CytoBrowser to find the images they need to have a z-level in the filename, if your image is only one 2D image (no z-stack), then just append `_z0` to the name, i.e., if your dzi output is named `image.dzi`, and the directory with all the patches `image_files`, then rename them to `image_z0.dzi` and `image_z0_files`. If you place these two in the `./data/` directory, then they should be found. 
In case you are running the Safari browser, do try with Chrome or Firefox, since Safari sometimes disallows local file acces.


I also suggest that you take a look at https://github.com/MIDA-group/CytoBrowser/blob/master/examples/Hamamatsu-OS-2.sh

If the above doesn't help, then you may wish to try to run the example script (requires wget and libvips; what's your platform?),
standing in the CytoBrowser directory run `. ./examples/Hamamatsu-OS-2.sh`

This will (you might grab a coffee while waiting):
1. download http://openslide.cs.cmu.edu/download/openslide-testdata/Hamamatsu/OS-2.ndpi (using wget, if you don't have it, you can download it manually);
2. run `vips` to convert it to dzi format, and giving it a name ending with '_z0' in the `./data`directory
3. and in the end try to open http://localhost:8080/?image=Hamamatsu-OS-2

I hope this helps.
Please let me know if it does, or otherwise, provide me with a bit more information on what you tried and what error message you face.
